### PR TITLE
Add labeled metrics to alarm system

### PR DESCRIPTION
**Pull request by hanahmily**
- Related issues

#5186

### CODE REVIEW

File: docs/en/setup/backend/backend-alarm.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ Alarm rule is constituted by following keys
 - **Exclude names**. The following entity names are excluded in this rule. Please follow [Entity name define](#entity-name).
 - **Include names regex**. Provide a regex to include the entity names. If both setting the include name list and include name regex, both rules will take effect.
 - **Exclude names regex**. Provide a regex to exclude the exclude names. If both setting the exclude name list and exclude name regex, both rules will take effect.
+- **Include labels**. The following labels of the metric are included in this rule.
+- **Exclude labels**. The following labels of the metric are excluded in this rule.
+- **Include labels regex**. Provide a regex to include labels. If both setting the include label list and include label regex, both rules will take effect.
+- **Exclude labels regex**. Provide a regex to exclude labels. If both setting the exclude label list and exclude label regex, both rules will take effect.
+*The settings of labels is required by meter-system which intends to store metrics from label-system platform, just like Prometheus, Micrometer, etc.
+The function supports the above four settings should implement `LabeledValueHolder`.*
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-alarm.md:32]: These seem not well-formated, please recheck.

File: oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java [hunk 2]
```diff
@@ -168,6 +161,46 @@ public void in(MetaInAlarm meta, Metrics metrics) {
         }
     }
 
+    private boolean validate(String target, List<String> includeList, List<String> excludeList,
```
Comment (wu-sheng) [inline oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java:164]: You are reusing this, please put comments on the method.

**Comment by wu-sheng** [comment]
Basically, good to me. Please fix some nits

File: docs/en/setup/backend/backend-alarm.md (hunk 1 above)
Comment (hanahmily) [reply docs/en/setup/backend/backend-alarm.md:33]: done

File: oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java (hunk 2 above)
Comment (hanahmily) [reply oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java:168]: done

**Pull request merged**